cloudify_handler/cloudify.py

```diff
@@ -29,10 +30,33 @@ def _bind(self):
            to set auto_delete=True)
         """
         credentials = pika.PlainCredentials(self.user, self.password)
+        try:
+            # If we're in an environment with celery we should have a
+            # worker configuration, but we'll check for backwards
+            # compatibility
+            import worker_conf
```

Comment:
worker_conf is the celery worker configuration deployed by either the bootstrap (for mgmt_worker) or cloudify agent installer.